### PR TITLE
[phoenix_v1.4.x] add new type definitions

### PR DESCRIPTION
**Pull request by hewsut**
- Links to documentation: https://phoenixframework.org
- Link to GitHub or NPM: https://github.com/phoenixframework/phoenix
- Type of contribution: addition

Other notes:
flowgen Typescript conversion

### CODE REVIEW

File: definitions/npm/phoenix_v1.4.x/flow_v0.25.x-/phoenix_v1.4.x.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
+// flow-typed signature: c03f8288d97b86dd25783ed76a0f1e28
+// flow-typed version: <<STUB>>/phoenix_v1.4.9/flow_v0.102.0
```
Comment (pascalduez) [inline definitions/npm/phoenix_v1.4.x/flow_v0.25.x-/phoenix_v1.4.x.js]: Could you please remove those comments?

Comment (hewsut) [reply definitions/npm/phoenix_v1.4.x/flow_v0.25.x-/phoenix_v1.4.x.js]: Sure thing. Done. If they aren't suppose to be there why does `flow-typed` generate them? 

This is my first contribution so my experience was:

1. `flow-typed` generated a stub for me (with those headers) and explicitly asked me to fill it out and contribute back.
2. I did.
3. Now I'm finding they shouldn't be there on submit.

Something feels off in that process.

**Comment by goodmind** [comment]
Nice! I tried converting myself and looks like some `declare type` were missing in initial conversion

File: definitions/npm/phoenix_v1.4.x/flow_v0.25.x-/phoenix_v1.4.x.js (hunk 1 above)
Comment (pascalduez) [reply definitions/npm/phoenix_v1.4.x/flow_v0.25.x-/phoenix_v1.4.x.js]: Those comment sort of make sense in your local `flow-typed` folder, when you stub a package. 
Not here on the main definitions repo.  
But I understand the feeling, following the path you describe.

**Pull request merged**